### PR TITLE
Add readme for Azure integration

### PR DESCRIPTION
**Pull request by narph**
## What does this PR do?

Add readme for Azure integration

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [x] I have verified that all datasets collect metrics or logs.

### CODE REVIEW

File: packages/azure/data_stream/activitylogs/manifest.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 type: logs
 title: Azure activitylogs logs
-release: experimental
+release: beta
```
Comment (kaiyan-sheng) [inline packages/azure/data_stream/activitylogs/manifest.yml:3]: Is this change intentional?

Comment (exekias) [reply packages/azure/data_stream/activitylogs/manifest.yml:3]: This sounds good to me, any opinion @sorantis?

File: dev/import-beats-resources/azure/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Azure Integration
+
+The azure integration retrieves different types of log data from Azure.
+There are several requirements before using the module since the logs will actually be read from azure event hubs.
+
+   - the logs have to be exported first to the event hubs https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create-kafka-enabled
```
Comment (exekias) [inline dev/import-beats-resources/azure/docs/README.md:6]: I think it should be singular?

```suggestion
   - the logs have to be exported first to the event hub https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create-kafka-enabled
```

**Review by exekias — APPROVED** [review]
